spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -913,30 +917,32 @@ def write_recording(recording: se.RecordingExtractor, save_path: PathType = None
             metadata = update_dict(recording.nwb_metadata, metadata)
 
         if nwbfile is None:
-            if Path(save_path).is_file():
+            if Path(save_path).is_file() and not overwrite:
                 read_mode = 'r+'
             else:
                 read_mode = 'w'
 
-            with NWBHDF5IO(save_path, mode=read_mode) as io:
+            with NWBHDF5IO(str(Path(save_path).absolute()), mode=read_mode) as io:
```

Comment:
Why do we need this?

Reply:
NWBHDF5IO doesn't support Path objects, only strings. But `spikeextractors` generally accepts `save_paths` to be Path objects, potentially.

Reply:
OK, but why are we casting as a `Path` and taking `absolute`? `NWBHDF5IO` can write to a relative path, can't it?

Reply:
It's casted since save_path can also optionally be a string to begin with. Suppose we don't need the absolute, though.

Reply:
I still do not understand why you would need to cast it as a `Path`

Reply:
We don't if we don't use the absolute anymore, removing